pyneuroml/plot/PlotSpikes.py

```diff
@@ -187,12 +185,54 @@ def read_sonata_spikes_hdf5_file(file_name: str):
     return ids_times_pops
 
 
-def run(a=None, **kwargs):
-    a = build_namespace(DEFAULTS, a, **kwargs)
-    logger.info(
-        "Generating spiketime plot for %s; format: %s; plotting: %s; save to: %s"
-        % (a.spiketime_files, a.format, a.show_plots_already, a.save_spike_plot_to)
-    )
+from typing import Dict, List, Optional, Union
```

Comment:
move these to the top of the file (imports should be at the top usually)

Hint: you can use something like [isort](https://pypi.org/project/isort/) to do this for you. It integrates with lots of IDEs, so check to see how to use it for your setup.gg